src/Source/DoctrineORMQuerySourceIterator.php

```diff
@@ -36,6 +41,8 @@ public function __construct(Query $query, array $fields, string $dateTimeFormat
             $this->query->setHint($name, $value);
         }
 
+        $this->batchSize = 100;
```

Comment:
What about adding the value to the constructor ?

Reply:
Ok, it's done !

Reply:
Hum do you think I should add it to `AbstractPropertySourceIterator` or not ?

Reply:
No because the ODM does not clear https://github.com/sonata-project/exporter/blob/2.x/src/Source/DoctrineODMQuerySourceIterator.php

Reply:
Yeah I was actually wondering if you would like to clear the ODM too ? But I won't be able to test the changes.